src/inmanta/ast/type.py

```diff
@@ -321,6 +324,9 @@ def __init__(self, basetype):
         Type.__init__(self)
         self.basetype = basetype
 
+    def normalize(self):
+        self.basetype.normalize
```

Comment:
missing brackets in function call
```suggestion
        self.basetype.normalize()
```